AssetTracking/AssetTracking/Extensions/AzureAdOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace AssetTracking.Extensions
+{
+    public class azureOptions
```

Comment:
Let's capitalize the first letter of this class name so that we maintain naming standards for Classes.